invokeai/backend/model_manager/load/model_loaders/flux.py

```diff
@@ -193,7 +198,30 @@ def _load_from_singlefile(
             sd = load_file(model_path)
             if "model.diffusion_model.double_blocks.0.img_attn.norm.key_norm.scale" in sd:
                 sd = convert_bundle_to_flux_transformer_checkpoint(sd)
+            futures: list[torch.jit.Future[tuple[str, torch.Tensor]]] = []
+            # For the first iteration we are just requesting the current size of the state dict
+            # This is due to an expected doubling of the tensor sizes in memory after converting float8 -> float16
+            # This should be refined in the future if not removed entirely when we support more data types
+            sd_size = asizeof.asizeof(sd)
+            cache_updated = False
+            for k in sd.keys():
+                v = sd[k]
+                if v.dtype != torch.bfloat16:
+                    if not cache_updated:
+                        self._ram_cache.make_room(sd_size)
+                        cache_updated = True
+                    futures.append(torch.jit.fork(convert_sd_entry_to_bfloat16, k, v))
```

Comment:
How much does using `torch.jit.fork()` speed this up? By running this asynchronously, we lose control over the peak memory utilization. In the worse case, we could hold the entire 8-bit and 16-bit versions of the model in memory simultaneously.